apps/src/applab/DesignToolbox.jsx

```diff
@@ -116,6 +117,14 @@ export default class DesignToolbox extends React.Component {
           elementType={'PHOTO_SELECT'}
           handleDragStart={this.props.handleDragStart}
         />
+        {experiments.isEnabled(experiments.APPLAB_ML) && (
```

Comment:
checking for an experiment flag in `render()` has caused performance issues in the past -- i'd move this check to `componentDidMount` and store the result in state